src/plugins/default/DuplicateCheckerPlugin.ts

```diff
@@ -1,50 +1,70 @@
 import { IInnerError } from '../../models/IInnerError';
-import { ILog } from '../../logging/ILog';
 import { IEventPlugin } from '../IEventPlugin';
 import { EventPluginContext } from '../EventPluginContext';
 import { Utils } from '../../Utils';
 
 export class DuplicateCheckerPlugin implements IEventPlugin {
-  public priority: number = 40;
+  public priority: number = 90;
   public name: string = 'DuplicateCheckerPlugin';
 
+  private _mergedEvents: MergedEvent[] = [];
   private _processedHashcodes: TimestampedHash[] = [];
   private _getCurrentTime: () => number;
+  private _interval: number;
 
-  constructor(getCurrentTime: () => number = () => Date.now()) {
+  constructor(getCurrentTime: () => number = () => Date.now(), interval: number = 60000) {
```

Comment:
there might be an event for that? I know there are libs that you can get notifications of when the page goes to sleep or isn't active, maybe we'd turn off queue processing / submit all events when that happens? Idk